Ginger/Ginger/Actions/ActionEditPages/ActSikuliEditPage.xaml.cs

```diff
@@ -110,6 +115,27 @@ private void SetJavaRelatedDetails()
             }
         }
 
+        private void ProcessValueTextBox_TextChanged(object sender, TextChangedEventArgs e)
+        {
+            if (!string.IsNullOrEmpty(xProcessValueEditor.ValueTextBox.Text))
+            {
+                ValueExpression mVE = new ValueExpression(Context.GetAsContext(actSikuli.Context).Environment, Context.GetAsContext(actSikuli.Context), WorkSpace.Instance.SolutionRepository.GetAllRepositoryItems<DataSourceBase>());
+                mVE.Value = xProcessValueEditor.ValueTextBox.Text;
+                string calculateValue = mVE.ValueCalculated;
+                actSikuli.ProcessNameVEForSikuliOperation = calculateValue;
+                for (int i = 0; i < xActiveProcessesTitlesComboBox.Items.Count; i++)
+                {
+                    ComboEnumItem item = xActiveProcessesTitlesComboBox.Items[i] as ComboEnumItem;
+                    if (item.Value.Equals(actSikuli.ProcessNameForSikuliOperation))
+                    {
+                        xActiveProcessesTitlesComboBox.SelectedIndex = i;
+                        break;
+                    }
+                }
+                xActiveProcessesTitlesComboBox.SelectedIndex = -1;
```

Comment:
Check if this line sets selectedindex to -1 even if selectedindex was set on line 131.